src/javascript/generateHook.mts

```diff
@@ -250,6 +250,18 @@ function generateHook(
                   ...options
                 }
                   );
+              };`;
+          result += `${hookName}.suspense = (
```

Comment:
You've already used the same pattern in prefetch(). Creating the same function again will double the file size. I thought to do it your way too but it's way too many edits because of the infinite query edits.

Reply:
Prefetch is not a hook, but suspense is a hook
you don't need to change whole of implementation, just change ``${hookName}.suspense = (` this line

Reply:
Alright i got you